modules/photosynthesis/R/plots.R

```diff
@@ -1,21 +1,26 @@
 ##' @title shaded confidence interval
 ##' @name ciEnvelope
 ##' @author Mike Dietze
-##' @export
+##' @noRd
 ciEnvelope <- function(x, ylo, yhi, col = "lightgrey", ...) {
-  return(polygon(cbind(c(x, rev(x), x[1]), c(ylo, rev(yhi), ylo[1])), 
+  return(graphics::polygon(cbind(c(x, rev(x), x[1]), c(ylo, rev(yhi), ylo[1])), 
                  col = col, border = NA, ...))
 } # ciEnvelope
 
 ##' @name plot_photo
 ##' @title plot_photo
 ##' @author Mike Dietze
+##' @param data input data
+##' @param out directory where output will be stored
```

Comment:
The parameter is misleadingly named, so I didn't catch this before: In this function `out` is the model output to be fitted, not an output directory.
```suggestion
##' @param out fitted model output from \code{\link{fitA}}
```